README.md

```diff
@@ -1,4 +1,4 @@
-## Welcome to GitHub Games
+## Welcome to awesome game of Tetris
 
 A project based learning activity for people who are getting started with Git and GitHub.
 
```

Comment:
> Thanks for opening this pull request, @tuves. 
> 
> ```suggestion
> You can play the game at: https://tuves.github.io/github-games/
> ```
> 
> I've made a suggested change to this pull request because I think you should link your version of the game in your README file. I made this suggestion by creating a review comment that uses some :sparkle: special :sparkle: formatting.
> 
> It looks like this:
> 
>     ```suggestion
>     You can play the game at: https://tuves.github.io/github-games/
>     ```
> 
> ### :keyboard: Activity: Apply the suggestion
> 
> 1. Click **Commit suggestion**
> 2. Enter a commit message
> 3. Click **Commit changes**
> 
> For more information about suggested changes, check out this [GitHub Help](https://help.github.com/articles/incorporating-feedback-in-your-pull-request) article.
> AZ